packages/block-editor/src/components/off-canvas-editor/style.scss

```
@@ -1 +1,15 @@
-//Styles for off-canvas editor, remove this line when you add some css to this file!
+.offcanvas-editor-list-view-tree-wrapper {
```

```suggestion
.offcanvas-editor-list-view-tree{
```

...in case you go with the above.